qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/AlternativeRouteActivity.kt

```diff
@@ -107,27 +108,11 @@ class AlternativeRouteActivity : AppCompatActivity(), OnMapLongClickListener {
         initListeners()
     }
 
-    override fun onStart() {
-        super.onStart()
-        binding.mapView.onStart()
```

Comment:
Is it safe to remove these? 👇

Reply:
Android Studio shows this warning:
<img width="592" alt="Screenshot 2021-09-22 at 21 55 46" src="https://user-images.githubusercontent.com/2395284/134406172-d118f9bd-f374-4147-bf22-1a8943b6b8e3.png">
And I checked that Maps actually calls these methods themselves.

Reply:
Thanks for confirming 🚀